### ISSUE TITLE
wdprops fails

### ISSUE THREAD

**Issue by Hunter-Github** [maxlath/wikidata-cli#2]
```
$ wdprops

Unhandled rejection Error: ENOENT: no such file or directory, open '~/.nodebrew/node/v5.11.1/props/node_modules/wikidata-cli/lib/en.json'
    at Error (native)

```

**Comment by maxlath** [comment]
thanks for reporting!
which version of wikidata-cli are you using? (that should be indicated in `~/.nodebrew/node/v5.11.1/props/node_modules/wikidata-cli/package.json` at "version")

**Comment by Hunter-Github** [comment]
1.7.1 at the moment.

**Comment by maxlath** [comment]
could you try installing the [debuggingIssue2](/maxlath/wikidata-cli/tree/debuggingIssue2) branch locally and see what it outputs?
```sh
git clone https://github.com/maxlath/wikidata-cli
cd wikidata-cli
git checkout debuggingIssue2
npm install
./bin/wdprops
```

**Comment by Hunter-Github** [comment]
Works fine now, even d/loads the necessary en.json.

**Comment by maxlath** [comment]
so the cloned version works but not the global package version?

**Comment by Hunter-Github** [comment]
1.7.2 fails to track the paths: when I run the global `wdprops`, it says no file at
~/.nodebrew/node/v5.11.1/**props/node_modules**/wikidata-cli/lib/en.json** - note that it's not the path where the installation is supposed to go. The local cloned version works all right.

**Comment by maxlath** [comment]
ok, I think 5d2ee23 should have fixed this issue, which was due to my lazyness to use a proper path resolution function, all my apologies for that ><
I published the patch so you should be able to test it by upgrading to the last version.

**Comment by Hunter-Github** [comment]
Definitely fixed in 1.7.5.
